invoices/invoiceregistry.go

```diff
@@ -56,6 +56,10 @@ type RegistryConfig struct {
 	// AcceptKeySend indicates whether we want to accept spontaneous key
 	// send payments.
 	AcceptKeySend bool
+
+	// KeySendHoldTime indicates for how long we want to accept and hold
+	// spontaneous key send payments.
```

Comment:
... before canceling them automatically?

also seems like `AcceptKeySend` is redundant now

Reply:
Not really, because how to otherwise configure keysend enabled without hold (immediate settle)?

Reply:
Ah I see, yes that case exists